ios/FluentUI/People Picker/Avatar.swift

```diff
@@ -28,6 +29,9 @@ open class AvatarData: NSObject, Avatar {
     public var secondaryText: String
     public var image: UIImage?
 
+    /// An image that can be used as a frame (outer wide border) for the avatar view
+    @objc public var customBorderImage: UIImage?
```

Comment:
nit: can we move the documentation up to the declaration of this in the protocol?